src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormattingContext.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
+{
+    /// <summary>
+    /// Represents an object containing the information to serialize with <see cref="IDistributedCacheTagHelperFormatter" />.
+    /// </summary>
+    public class DistributedCacheTagHelperFormattingContext
```

Comment:
I don't mind this class, though I would take it a step further and consider putting the Key, Html, and Data all on the context. SerializeAsync and DeserializeAsync could then ship Html->Data and Html<-Data respectively, and could stash and verify the Key inside the Data as well. Though that last bit is more a part of the next issue dealing with making a proper key class instead of doing string manipulation.